app/shared/actions/blockexplorers.js

```diff
@@ -0,0 +1,63 @@
+import sortBy from 'lodash/sortBy';
+
+import * as types from './types';
+import eos from './helpers/eos';
+
+export function getBlockExplorers() {
+  return (dispatch: () => void, getState) => {
+    dispatch({
+      type: types.SYSTEM_BLOCKEXPLORERS_PENDING
+    });
+    // const { connection } = getState();
```

Comment:
I left these lines commented out as they will be used when we integrate the smart-contract.